### ISSUE TITLE
what is the difference between num_procs==0 and the metric being absent

### ISSUE THREAD

**Issue by jumbovin** [ncabatoff/process-exporter#133]
I see few metrics listed with 0 num_procs value for a longer duration while some of the metrics become absent, what is different in both these case.
I dont see any processes running under the groupname even wither if it is num_proces or the metric is absent

**Comment by rntbrgtt** [comment]
+ 1!
I would like to see all procs listed on conf file showing on the exporter, even the ones with num_procs=0. If they are listed and are not running I need to see it.

**Comment by LeoHsiao1** [comment]
In my opinion:
When process-exporter monitors process A, it records num_procs=1.
If process A disappears, it will record num_procs=0.
If process-exporter restarts, it will only record the processes that exist at this time, not process A.

**Comment by rntbrgtt** [comment]
Yes that's the way it's working for me but if I am monitoring process A and this is not running I would like to see that.

**Comment by Gourds** [comment]
+1 same question
